test/unit/lock.spec.js

```diff
@@ -0,0 +1,60 @@
+const Lock = require("../../src/lock");
+
+describe("Test lock", () => {
+  let key = "abc123"
+  it("should lock", () => {
+    const lock = new Lock();
+
+    return lock.acquire(key).then(() => {
+      expect(lock.isLocked(key)).toBeTruthy()
+      lock.release(key).then(() => {
```

Comment:
missing `return` from the start of the line. It's important because if an error occurs, jest won't detect it and it will say the test is passed.

Reply:
OK, I will fix it.